### DISCUSSION TITLE
ELK : Logs issues - Unable to see the log in the 'tail -f /var/log/messages'

### DISCUSSION THREAD

**Discussion by gkalluru77** [learndevopsonline/learndevopsonline#1610] [Q&A] [answered]
Have configured ELK and Kibana and Logstash. and trying to hit some events nothing is generated in messages logs please see the generated output in the below

Public IPs: 
workstation: 3.90.173.67
elk:  3.90.173.67
Frontend: 54.162.87.76
backend: 54.87.160.196
mysql : 34.207.213.209

54.162.87.76 | 172.31.40.215 | t3.micro | null
[ root@frontend ~ ]# tail -f /var/log/messages
Nov 16 11:30:11 ip-172-31-40-215 systemd[12101]: Reached target Shutdown.
Nov 16 11:30:11 ip-172-31-40-215 systemd[12101]: Started Exit the Session.
Nov 16 11:30:11 ip-172-31-40-215 systemd[12101]: Reached target Exit the Session.
Nov 16 11:30:11 ip-172-31-40-215 systemd[1]: user@0.service: Succeeded.
Nov 16 11:30:11 ip-172-31-40-215 systemd[1]: Stopped User Manager for UID 0.
Nov 16 11:30:11 ip-172-31-40-215 systemd[1]: Stopping User runtime directory /run/user/0...
Nov 16 11:30:11 ip-172-31-40-215 systemd[1]: run-user-0.mount: Succeeded.
Nov 16 11:30:11 ip-172-31-40-215 systemd[1]: user-runtime-dir@0.service: Succeeded.
Nov 16 11:30:11 ip-172-31-40-215 systemd[1]: Stopped User runtime directory /run/user/0.
Nov 16 11:30:11 ip-172-31-40-215 systemd[1]: Removed slice User Slice of UID 0.

**Comment by sreedharm07** [comment]
Hi @gkalluru77 
nginx logs will be in  /var/log/nginx/access.log file. and there is logs 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/ccccb486-ce07-4183-aeb9-c8517dffa9ed)

**Reply by gkalluru77** [reply]
Today session Raghu opened the logs command one which i given

**Reply by gkalluru77** [reply]
and not able to see the log in Kibana UI

**Reply by gkalluru77** [reply]
I doubt something went wrong in the filebeat.yml configuration file

**Reply by gkalluru77** [reply, marked as answer]
@sreedharm07 

It resolved... just executed the same no changes...


54.162.87.76 | 172.31.40.215 | t3.micro | null
[ root@frontend ~ ]# tail -f /var/log/messages
Nov 16 12:20:01 ip-172-31-40-215 systemd[15426]: Reached target Timers.
Nov 16 12:20:01 ip-172-31-40-215 systemd[15426]: Reached target Paths.
Nov 16 12:20:01 ip-172-31-40-215 systemd[15426]: Listening on D-Bus User Message Bus Socket.
Nov 16 12:20:01 ip-172-31-40-215 systemd[15426]: Reached target Sockets.
Nov 16 12:20:01 ip-172-31-40-215 systemd[15426]: Reached target Basic System.
Nov 16 12:20:01 ip-172-31-40-215 systemd[15426]: Reached target Default.
Nov 16 12:20:01 ip-172-31-40-215 systemd[15426]: Startup finished in 32ms.
Nov 16 12:20:01 ip-172-31-40-215 systemd[1]: Started User Manager for UID 0.
Nov 16 12:20:01 ip-172-31-40-215 systemd[1]: Started Session 46 of user root.
Nov 16 12:20:01 ip-172-31-40-215 systemd[1]: session-46.scope: Succeeded.
Nov 16 12:20:10 ip-172-31-40-215 filebeat[15376]: {"log.level":"info","@timestamp":"2023-11-16T12:20:10.079Z","log.logger":"monitoring","log.origin":{"file.name":"log/log.go","file.line":187},"message":"Non-zero metrics in the last 30s","service.name":"filebeat","monitoring":{"metrics":{"beat":{"cgroup":{"memory":{"mem":{"usage":{"bytes":96919552}}}},"cpu":{"system":{"ticks":50},"total":{"ticks":180,"time":{"ms":10},"value":180},"user":{"ticks":130,"time":{"ms":10}}},"handles":{"limit":{"hard":262144,"soft":262144},"open":12},"info":{"ephemeral_id":"c69d55d3-cfbd-4f19-b967-e5c5dc85d735","uptime":{"ms":150130},"version":"8.11.1"},"memstats":{"gc_next":36630256,"memory_alloc":18041048,"memory_total":67281392,"rss":116449280},"runtime":{"goroutines":32}},"filebeat":{"events":{"active":0},"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"output":{"events":{"active":0}},"pipeline":{"clients":1,"events":{"active":0}}},"registrar":{"states":{"current":0}},"system":{"load":{"1":0.09,"15":0.01,"5":0.04,"norm":{"1":0.045,"15":0.005,"5":0.02}}}},"ecs.version":"1.6.0"}}
Nov 16 12:20:11 ip-172-31-40-215 systemd[1]: Stopping User Manager for UID 0...
Nov 16 12:20:11 ip-172-31-40-215 systemd[15426]: Stopped target Default.
Nov 16 12:20:11 ip-172-31-40-215 systemd[15426]: Stopped target Basic System.
Nov 16 12:20:11 ip-172-31-40-215 systemd[15426]: Stopped target Sockets.
Nov 16 12:20:11 ip-172-31-40-215 systemd[15426]: Stopped target Paths.
Nov 16 12:20:11 ip-172-31-40-215 systemd[15426]: Stopped target Timers.
Nov 16 12:20:11 ip-172-31-40-215 systemd[15426]: Closed D-Bus User Message Bus Socket.
Nov 16 12:20:11 ip-172-31-40-215 systemd[15426]: Reached target Shutdown.
Nov 16 12:20:11 ip-172-31-40-215 systemd[15426]: Started Exit the Session.
Nov 16 12:20:11 ip-172-31-40-215 systemd[15426]: Reached target Exit the Session.
Nov 16 12:20:11 ip-172-31-40-215 systemd[1]: user@0.service: Succeeded.
Nov 16 12:20:11 ip-172-31-40-215 systemd[1]: Stopped User Manager for UID 0.
Nov 16 12:20:11 ip-172-31-40-215 systemd[1]: Stopping User runtime directory /run/user/0...
Nov 16 12:20:11 ip-172-31-40-215 systemd[1]: run-user-0.mount: Succeeded.
Nov 16 12:20:11 ip-172-31-40-215 systemd[1]: user-runtime-dir@0.service: Succeeded.
Nov 16 12:20:11 ip-172-31-40-215 systemd[1]: Stopped User runtime directory /run/user/0.
Nov 16 12:20:11 ip-172-31-40-215 systemd[1]: Removed slice User Slice of UID 0.
Nov 16 12:20:19 ip-172-31-40-215 filebeat[15376]: {"log


![image](https://github.com/learndevopsonline/learndevopsonline/assets/149259316/13bb7d50-158b-4a22-a461-40171bb693d8)
